cognite/extractorutils/rest/http.py

```diff
@@ -34,6 +34,7 @@
     Dict[str, "RequestBodyTemplate"],  # type: ignore
     Callable[[], Dict[str, "RequestBodyTemplate"]],  # type: ignore
 ]
+JsonType = Union[str, int, float, bool, Dict, List]
```

Comment:
I think this is what the `RequestBody` type was meant to be. Maybe rather update the name of that type to `JsonBody` or something like that instead of making this one?